helm/policies-azure/Chart.yaml

```diff
@@ -1,9 +1,11 @@
+annotations:
```

Comment:
I think you need to revert this file. Sometimes happens with `abs` :/

Reply:
done